src/store/plugins/veeValidate.js

```diff
@@ -112,11 +112,9 @@ export default (store) => {
   defineRule('name_unregistered', (value) => checkName(NAME_STATES.UNREGISTERED)(`${value}.chain`, []));
   defineRule('name_registered_address', (value) => checkAensName(value) && checkNameRegisteredAddress(value));
   defineRule('token_to_an_address',
-    (value, [isToken]) => (
-      !checkAensName(value)
-      || (checkAensName(value) && !isToken)
-    ),
-    { params: ['isToken'] });
+    (value, [isToken]) => !checkAensName(value) || (checkAensName(value) && !isToken), {
+      params: ['isToken'],
+    });
```

Comment:
I feel that the previous formatting of those lines was clearer

Reply:
Actually that change was done by npm run lint:vue:fix because error on serve/build existed.